blackbox/docs/general/builtins/window-functions.rst

```diff
@@ -0,0 +1,69 @@
+.. highlight:: psql
+.. _window-functions:
+
+================
+Window Functions
+================
+
+.. rubric:: Table of Contents
+
+.. contents::
+   :local:
+
+Introduction
+============
+
+:ref:`Aggregation <aggregation>` functions can be executed over a subset of the
```

Comment:
I think we should introduce window functions by explaining their behaviour first before mentioning that aggregation functions can be used as window functions.

E.g.:

Window functions are functions which perform some sort of computation across a set of rows which are related to the current row. This is comparable to aggregation functions, but window functions do not cause multiple rows to become grouped into a single row.